sdk/core/allocator/alloc.h

```diff
@@ -289,7 +289,13 @@ namespace displacement_proxy
 struct __packed __aligned(MallocAlignment)
 MChunkHeader
 {
-	static constexpr size_t OwnerIDWidth = 14;
+	/**
+	 * Each chunk has a 16-bit metadata field that is used to store a small
+	 * bitfield and the owner ID in the remaining bits.  This is the space not
+	 * consumed by the metadata.  It must be reduced if additional bits are
+	 * stolen for other fields.
+	 */
+	static constexpr size_t OwnerIDWidth = 13;
```

Comment:
For a follow-up PR, perhaps: do we ever actually check that the `uint16_t` owner IDs we're passing around fit in this width?  Maybe just in debug builds?

Reply:
In fact, we do!

https://github.com/microsoft/cheriot-rtos/blob/8fc5d7da8fd15731580fc8031dc73b65ebc0e581/sdk/core/allocator/main.cc#L358